pkg/controller.v2/controller_pod.go

```diff
@@ -212,9 +198,9 @@ func getPodSlices(pods []*v1.Pod, replicas int, logger *log.Entry) [][]*v1.Pod {
 		}
```

Comment:
Should we break here?

Reply:
Sorry, I rebased the master and can not see it anymore, could you have another review and point it out?

Reply:
It has already been addressed.